pyccel/codegen/printing/pycode.py

```diff
@@ -418,6 +418,22 @@ def _print_NumpyAutoFill(self, expr):
                 dtype = dtype,
                 order = expr.order)
 
+    def _print_NumpyLinspace(self, expr):
+        type_name = type(expr).__name__
+        func_name = type_name[5:].lower()
+        return "{func_name}({start}, {stop}, {size})".format(
+                func_name = func_name,
+                start = self._print(expr.start),
+                stop = self._print(expr.stop),
+                size = self._print(expr.size))
+
+    def _print_NumpyMatmul(self, expr):
+        return "{func_name}({x1}, {x2})".format(
+                func_name = 'matmul',
```

Comment:
Does this need to be a parameter? `"matmul({x1}, {x2})".format(..)` is faster for python